ingeniamotion/wizard_tests/feedbacks_tests/feedback_test.py

```diff
@@ -100,9 +97,26 @@ def __init__(self, mc: "MotionController", servo: str, axis: int) -> None:
         self.pos_vel_same_feedback = False
         self.resolution_multiplier = 1.0
         self.test_frequency = self.TEST_FREQUENCY
+        self.add_registers_to_backup()
         self.backup_registers_names = self.BACKUP_REGISTERS
         self.suggested_registers = {}
 
+    def add_registers_to_backup(self) -> None:
```

Comment:
This function is not necessary. See the comment above.

Reply:
This function is used to ignore registers that are not needed in an active way. I think it's clearer from conceptual point of view in this way. What do you think?

Reply:
The thing is sometimes there are drives that they don't need some registers to run a test. That means these registers are not in the dictionary, thus, a log warning will be raised and it seems something is going badly.

In order to differentiate what is a mistake and what is not a mistake, we conclude to add optional registers to append them to the backup registers list for these tests.
If an optional register exists means it's important for a proper test, instead the test can be run without any problem.

This logic will be moved to the base test class, and the optional registers will be selected in the child classes.